python/ray/worker.py

```diff
@@ -468,6 +468,21 @@ def get_object(self, object_ids):
 
         if len(unready_ids) > 0:
             with self.state_lock:
+                # Get the task ID, to notify the backend which task is blocked.
+                current_task_id = self.current_task_id
+                if not ray.utils.is_main_thread():
```

Comment:
Consider moving this into a helper method

Reply:
Done.